docs/metrics-ndm.md

```diff
@@ -0,0 +1,24 @@
+# Monitoring node-disk-manager(NDM)
+
+## Metrics exported by ndm cluster exporter as of the current release are
+
+```
+node_block_device_state # State of BlockDevice (0,1,2) = {Active, Inactive, Unknown}
```

Comment:
Graph shows 0.5 and 1.5 as well, can those be hidden?

Reply:
No, that can't be removed. One way would be to decrease the panel size then that will not be seen but if someone extends the panel again it will be visible. And if we decrease the size then panel information is not properly visible.